app/controllers/application_controller.rb

```diff
@@ -1,2 +1,5 @@
 class ApplicationController < ActionController::Base
+  def current_user
+    User.first
```

Comment:
- [ ] Kindly allow users to add likes to posts as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/forms_project_v1_1.md#:~:text=Allow%20Users%20to%20add%20likes%20to%20Posts.).

Reply:
thank you, I add it